byte-ranges-specifier value  1900- explained (#20479)
mdn/content: files/en-us/web/http/headers/range/index.md

> ```http

Reviewer: ```suggestion

```http
```

Range: bytes=200-1000, 2000-6576, 19000-
```

> The byte-ranges-specifier value `19000-` is using the suffix-byte-range-spec  `-` to indicates that all bytes from 19000 onwards shall be part of the third range. 

Reviewer: ```suggestion
The ranges-specifier value `19000-` specifies `19000` as the first position, and omits any last position — in order to indicate that all bytes from 19000 onward are part of the third range. 
```

> The ranges-specifier value `19000-` specifies `19000` as the first position, and omits any last position — in order to indicate that all bytes from 19000 onward are part of the third range. 

Reviewer: ```suggestion

The ranges-specifier value `19000-` specifies `19000` as the first position, and omits any last position — in order to indicate that all bytes from 19000 onward are part of the third range. 
```
